app/services/make_an_offer.rb

```diff
@@ -39,4 +45,14 @@ def validate_offer_conditions
     errors.add(:offer_conditions, "has over #{MAX_CONDITIONS_COUNT} elements") if @offer_conditions.count > MAX_CONDITIONS_COUNT
     errors.add(:offer_conditions, "has a condition over #{MAX_CONDITION_LENGTH} chars in length") if @offer_conditions.any? { |c| c.length > MAX_CONDITION_LENGTH }
   end
+
+  def set_decline_by_default
+    days = TimeLimitCalculator.new(
+      rule: :decline_by_default,
+      effective_date: Time.zone.now,
+      ).call
+
+    application_choice.decline_by_default_days = days
+    application_choice.decline_by_default_at = days.business_days.from_now.end_of_day
+  end
```

Comment:
Is this method going to be replaced by a call to `SetDeclineByDefault`?

Reply:
Yes! we'll call the service.